keyboards/tkc1800/rules.mk

```diff
@@ -9,12 +9,13 @@ MCU = at90usb1286
 #   QMK DFU      qmk-dfu
 #   ATmega32A    bootloadHID
 #   ATmega328P   USBasp
-BOOTLOADER = atmel-dfu
+BOOTLOADER = qmk-dfu
```

Comment:
I'm fine with the keymap changes. This looks like a much cleaner way to disable i2c, which is incompatible with console logging if you didn't have the OLED.

My only concern here is that all the older PCBs in existence, are still be running atmel-dfu, which has a slightly different behavior around erasure. Usually how this handled by PCB rev folders. I haven't tested what happens if I try to RESET with the bootloader set to qmk-dfu instead of atmel-dfu.

I can certainly add a rules.mk override so this isn't a blocker, more of a technical question. I can also ISP flash my pcb and add qmk-dfu.

Reply:
@yanfali I've tested it, works without issue. Having the QMK code setup for qmk-dfu works correctly whether the bootloader is qmk-dfu or atmel-dfu.